libsolutil/Whiskers.cpp

```diff
@@ -74,6 +75,17 @@ string Whiskers::render() const
 	return replace(m_template, m_parameters, m_conditions, m_listParameters);
 }
 
+void Whiskers::checkTemplateValid() const
+{
+	regex validTemplate("<[?!\\/][a-zA-Z0-9_$-]+(?:[^a-zA-Z0-9_$>-]|$)");
```

Comment:
> 	/// Sets a list parameter, <#listName> </listName>.

There are also list parameters which could be truncated. Perhaps should support them here.

Reply:
Sure, I will include the check for list params